### PR TITLE
EZP-30346: Broken notifications with html

### PR DESCRIPTION
**Pull request by ViniTou**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-30346
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | yes
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/170

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/admin.notifications.js [hunk 1]
```diff
@@ -3,14 +3,19 @@
     const notifications = JSON.parse(notificationsContainer.dataset.notifications);
     const template = notificationsContainer.dataset.template;
     const addNotification = ({ detail }) => {
-        const { onShow, label, message } = detail;
+        const { onShow, label, message, rawPlaceholdersMap } = detail;
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.notifications.js]: Probably will fail if backend notification occurs. It needs the default value `{}`

**Comment by dew326** [comment]
Open question: Shouldn't this part go to 1.3 as the issue was targeted 1.3 and the problem is there? But on the other hand, we don't have HTML in notifications in 1.3. cc @lserwatka

**Comment by lserwatka** [comment]
No, if we don't have it in 1.3, we should go to 1.4.

**Comment by lserwatka** [comment]
@ViniTou what is needed here?

**Comment by ViniTou** [comment]
@lserwatka 
@dew326 

final round of review (I have fixed default value) and QA.

**Comment by lserwatka** [comment]
Could you merge it up?

**Pull request merged**